airbyte-api/src/main/openapi/config.yaml

```diff
@@ -2871,6 +2917,33 @@ components:
       properties:
         consentUrl:
           type: string
+    OAuthSourceRequestBody:
+      type: object
+      required:
+        - authCode
+        - sourceDefinitionId
+      properties:
+        authCode:
+          type: string
+        sourceDefinitionId:
+          $ref: "#/components/schemas/SourceDefinitionId"
+    OAuthDestinationRequestBody:
+      type: object
+      required:
+        - authCode
+        - destinationDefinitionId
+      properties:
+        authCode:
+          type: string
+        destinationDefinitionId:
+          $ref: "#/components/schemas/DestinationDefinitionId"
+    OAuthRead:
```

Comment:
I think this is going to be connector specific. Sometimes there'll be refreshtoken, others refreshToken + accessToken, others accessToken etc... and their names may be slightly different. we should potentially rely on additionalProperties too